features/bevragen/persoon/verblijfplaats/locatie/in-onderzoek.feature

```diff
@@ -0,0 +1,192 @@
+    | naam                        | waarde                      |
+    | gemeentecode (92.10)        | 0518                        |
+    | locatiebeschrijving (12.10) | Woonboot bij de Grote Sloot |
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                               |
+    | type                | RaadpleegMetBurgerservicenummer      |
+    | burgerservicenummer | 000000152                            |
+    | fields              | verblijfplaats.datumIngangGeldigheid |
+    Dan heeft de response een persoon met de volgende 'verblijfplaats' gegevens
+    | naam                                         | waarde      |
+    | type                                         | Locatie     |
+    | inOnderzoek.type                             | <type io>   |
+    | inOnderzoek.datumIngangGeldigheid            | true        |
+    | inOnderzoek.datumIngangOnderzoek.type        | Datum       |
+    | inOnderzoek.datumIngangOnderzoek.datum       | 2002-07-01  |
+    | inOnderzoek.datumIngangOnderzoek.langFormaat | 1 juli 2002 |
+
+    Voorbeelden:
+    | aanduiding in onderzoek | type io |
+    | 080000                  | true    |
+    | 088500                  |         |
+    | 088510                  |         |
+
+  Abstract Scenario: 'functie adres (10.10)' is in onderzoek, en wordt gevraagd
+    Gegeven de persoon met burgerservicenummer '000000152' heeft de volgende 'verblijfplaats' gegevens
+    | naam                            | waarde                    |
+    | aanduiding in onderzoek (83.10) | <aanduiding in onderzoek> |
+    | datum ingang onderzoek (83.20)  | 20020701                  |
+    En de 'verblijfplaats' heeft de volgende 'adres' gegevens
+    | naam                        | waarde                      |
+    | gemeentecode (92.10)        | 0518                        |
+    | locatiebeschrijving (12.10) | Woonboot bij de Grote Sloot |
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000152                       |
+    | fields              | verblijfplaats.functieAdres     |
+    Dan heeft de response een persoon met de volgende 'verblijfplaats' gegevens
+    | naam                                         | waarde      |
+    | type                                         | Locatie     |
+    | inOnderzoek.type                             | <type io>   |
+    | inOnderzoek.functieAdres                     | true        |
+    | inOnderzoek.datumIngangOnderzoek.type        | Datum       |
+    | inOnderzoek.datumIngangOnderzoek.datum       | 2002-07-01  |
+    | inOnderzoek.datumIngangOnderzoek.langFormaat | 1 juli 2002 |
+
+    Voorbeelden:
+    | aanduiding in onderzoek | type io |
+    | 080000                  | true    |
+    | 081000                  |         |
+    | 081010                  |         |
+
```

Comment:
Ik weet niet of je in dit feature dezelfde soort scenario functionaliteit wil afdekken als in 'features\bevragen\persoon\verblijfplaats\adres\in-onderzoek.feature' maar als je dat wil dan mist hier nog het scenario uit regel 68 in dat feature bestand ('gemeentedeel' is in onderzoek, maar is geen veld van verblijfplaats binnenland (adres) resource).